cmd/thanos/receive.go

```diff
@@ -144,6 +143,9 @@ func runReceive(
 		return err
 	}
 
+	// Has this thanos receive instance been configured to ingest metrics into a local TSDB?
+	enableIngestion := receiveMode == receive.IngestorOnly || receiveMode == receive.RouterIngestor
```

Comment:
This is used only once? If yes, let's just inline WDYT?

Reply:
It is actually used twice [here](https://github.com/thanos-io/thanos/blob/be9b0f5d85a23795403c60aba96e1eb22073a1bc/cmd/thanos/receive.go#L150) & [here](https://github.com/thanos-io/thanos/blob/be9b0f5d85a23795403c60aba96e1eb22073a1bc/cmd/thanos/receive.go#L225)